Transport/Ftp/FtpTransport.php

```diff
@@ -94,7 +95,7 @@ public function reject(Envelope $envelope): void
     public function send(Envelope $envelope): Envelope
     {
         $this->logger->debug('send called');
-        
-        return $envelope;
+
+        throw Exception('Problème to send on your producer');
```

Comment:
Non il n'y a pas de "problem". C'est juste non supporté ^^

Plutôt mettre "This transport does not support the send action"